_includes/assets/sass/components/_home-page.scss

```diff
@@ -0,0 +1,27 @@
+.measure {
+  @media (min-width: $tbds-breakpoint-medium) {
+    max-width: 60ch;
+  }
+
+  @media (min-width: $tbds-breakpoint-medium) {
+    max-width: 70ch;
+  }
+}
+
+.measure-large {
```

Comment:
We can remove this since the media query above is already accomplishing the same thing.